crates/bevy_ecs/src/entity/hash.rs

```diff
@@ -0,0 +1,95 @@
+use std::hash::{BuildHasher, Hasher};
+
+use bevy_reflect::Reflect;
```

Comment:
This (and all its uses) should be present only if the `bevy_reflect` feature of the `bevy_ecs` crate is enabled, otherwise the import won't be found.